src/react-components/src/components/01-atoms/Forms/Input.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import Input from "./Input";
+
+export default {
+  title: "Input",
+  component: Input,
+};
+
+export const input = () => <Input labelId="blah"
+  labelContent={<>Label</>}
+  inputId="blah"
+  isRequired={false}
+></Input>;
+
+export const inputWithHelper = () => <Input labelId="blah"
+  labelContent={<>Label</>}
+  inputId="blah"
+  helperContentOpts={{id: "blah", content: <>This is some helpful text</>}}
+/>;
+export const inputRequiredWithError = () => <Input labelId="blah"
+  labelContent={<>Label</>}
+  inputId="blah"
+  isRequired={true}
+  errorContentOpts={{id: "blah", content: <>Error Message Lorem Ipsum</>}}
+  showError={true}></Input>;
+
+export const inputRequiredWithHelperAndError = () => <Input labelId="blah"
```

Comment:
I think we should delete this story... in fact I don't think you should be able to show helper AND error text at the same time, I don't think it makes sense. That might be an error we want to throw unless you think I'm thinking about this wrong

Reply:
Since dropdown currently also can have both helper and error, I think we should split this out into a separate task.  I'm not sure where that should live - I'm happy to make a ResearchNow ticket for it but I don't know how it will be prioritized.  

Mainly, I think this will require a re-thinking of the way HelperErrorText is done - HelperErrorText can't really check number of instances of itself (nor should it, DateRangeForm has two helpers) so I think we might want to change it to a wrapper or HOC that appends helper or error based on certain checks.  

If we can, I would rather not block this ticket pending that work.

Reply:
That makes sense. Let's ask for a backlog ticket from Andrew in SFR (since this does get shipped in RN and I think the current implementation is kind of wonky I think it's fair to ask for).